### ISSUE TITLE
Users report the hamburger icon is not tapable

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#2157]
Happens at least on a LG Optimus G and a Huawei Ascend G510

**Comment by roundhill** [comment]
FWIW the drawer opens on my 4.0.3 Samsung Galaxy device.

**Comment by maxme** [comment]
List of devices affected by this issue:
```
LG Optimus G
LG Optimus F6 
HUAWEI G525-U00 
HUAWEI G510
HUAWEI P6 
HUAWEI MediaPad X1 7.0 
```

**Comment by maxme** [comment]
fixed via #2190
